src/components/AppProvider/types.ts

```diff
@@ -31,18 +29,9 @@ export interface AppProviderProps {
 }
 
 export interface Context {
-  polaris: {
```

Comment:
Removing the `polaris` namespace now since we consume it from a render prop rather than context types. We won't have to worry about collisions now